config/v1/types_cluster_operator.go

```diff
@@ -143,48 +143,49 @@ type ClusterOperatorStatusCondition struct {
 type ClusterStatusConditionType string
 
 const (
-	// Available indicates that the operand (eg: openshift-apiserver for the
```

Comment:
In case it helps with review, I personally find the output of:

```console
$ git --no-pager show --word-diff=color
```

easier to read for this commit than GitHub's rendering.